backend/api/Services/DocumentService.cs

```diff
@@ -50,5 +56,10 @@ public async Task<ExternalResult<DocumentDetail>> UploadDocumentAsync(int docume
             task.Wait();
             return task.Result;
         }
+
+        public IEnumerable<PimsDocumentTyp> GetPIMSDocumentTypes()
+        {
+            return documentTypeRepository.GetAll();
+        }
```

Comment:
If the types are returned as part of the lookup service, is this needed?

Reply:
Please refer to https://github.com/bcgov/PSP/pull/1760#discussion_r925116695